doc/cephfs/fs-nfs-exports.rst

```diff
@@ -36,6 +36,15 @@ Update NFS Ganesha Cluster
 
 This updates the deployed cluster according to the placement value.
 
+Delete NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster delete <type=cephfs> <clusterid>
```

Comment:
Again, not sure what value type=cephfs adds here. It seems like you should really only have to specify the type= parameter when you're creating it.